client/rpc/src/eth/filter.rs

```diff
@@ -31,11 +35,6 @@ use sp_runtime::{
 	generic::BlockId,
 	traits::{Block as BlockT, NumberFor, One, Saturating, UniqueSaturatedInto},
 };
-// Frontier
-use fc_rpc_core::{types::*, EthFilterApiServer};
-use fp_rpc::{EthereumRuntimeRPCApi, TransactionStatus, TxPoolRuntimeApi};
-
-use crate::{eth::cache::EthBlockDataCacheTask, frontier_backend_client, internal_err, TxPool};
```

Comment:
I recommend using `StdExternalCrate` to format the import order. 
For `External` imports, adding some comments to distinguish whether they originate from substrate/frontier.

like below:

```rust
use std::{collections::HashSet, marker::PhantomData, sync::Arc, time};

use ethereum::BlockV2 as EthereumBlock;
use ethereum_types::{H256, U256};
use jsonrpsee::core::{async_trait, RpcResult};
// Substrate
use sc_client_api::backend::{Backend, StorageProvider};
use sc_transaction_pool::ChainApi;
use sp_api::ProvideRuntimeApi;
use sp_blockchain::{Error as BlockChainError, HeaderBackend, HeaderMetadata};
use sp_core::hashing::keccak_256;
use sp_runtime::{
	generic::BlockId,
	traits::{Block as BlockT, NumberFor, One, Saturating, UniqueSaturatedInto},
};
// Frontier
use fc_rpc_core::{types::*, EthFilterApiServer};
use fp_rpc::{EthereumRuntimeRPCApi, TransactionStatus, TxPoolRuntimeApi};

use crate::{eth::cache::EthBlockDataCacheTask, frontier_backend_client, internal_err, TxPool};
```

Reply:
The best option is to use rustfmt's `group_imports = "StdExternalCrate"` configuration directly, but it does not support subdividing `External` crates using comments like above, so we can only manually sort them temporarily.

Reply:
Okay. I'll submit a dedicated commit to keep the style consistent with other frontier crates.